automation_tools/__init__.py

```diff
@@ -2257,7 +2257,7 @@ def product_install(distribution, create_vm=False, certificate_url=None,
 
     # enable cockpit feature
     if (sat_version not in ('6.3', '6.4', '6.5', '6.6')):
```

Comment:
need to fix this condition as well, for upstream-nightly we already have added` enable-foreman-plugin-remote-execution-cockpit` option in `def katello_installer.`

Reply:
> need to fix this condition as well, for upstream-nightly we already have added` enable-foreman-plugin-remote-execution-cockpit` option in `def katello_installer.`

i removed the line from the `katello_installer` method, would that work for  you?